pkg/operator/secretannotator/status/status.go

```diff
@@ -16,36 +16,46 @@ import (
 	"github.com/openshift/cloud-credential-operator/pkg/util/clusteroperator"
 )
 
+type SecretStatusHandler struct {
+	client.Client
```

Comment:
Name this field. I don't see any reason why `SecretStatusHandler` should be inheriting from `client.Client` instead of just composing it.